docs/src/main/sphinx/connector/hive.rst

```diff
@@ -778,6 +778,10 @@ connector.
       - The target number of buffered splits for each table scan in a query,
         before the scheduler tries to pause.
       - ``1000``
+    * - ``hive.max-outstanding-splits-size``
+      - The maxiumum size allowed for buffered splits for each table scan
+        in a query, before the query fails.
+      - ``256 MB``
```

Comment:
no space in these values allowed I am pretty sure

Reply:
@mosabua Are you referring to the "256 MB"? Other file size values in the same table have a space there, such as hive.max-initial-split-size and hive.max-split-size, which have "32 MB" and "64 MB" as values, respectively